docs/scaffold-a-new-project/README.md

```diff
@@ -0,0 +1,72 @@
+presets behind the scene to build projects. If you are not familiarized with them, take a moment to
+explore [what presets are](https://neutrino.js.org/presets/).
+
+## Getting Started
+Run the following command to start the interactive session. Substitute `<directory-name>` with the directory name you wish to create for this project.
+
+### Yarn
+
+```
+❯ yarn create neutrino-project <directory-name>
+```
+
+_Note: The `create` command is a shorthand that helps you do two things at once. See the [Yarn create docs](https://yarnpkg.com/lang/en/docs/cli/create/) for more details._
+
+### npm
+
+```
+❯ npm install -g create-neutrino-project
+❯ create-neutrino-project <directory-name>
+```
+
+## Available Projects
+`create-neutrino-project` presently offers a scaffolding project to build an application, a library,
+or components. Depending on the project type, the CLI helper may offer different flavors
+of that project to scaffold. Each project type harnesses the power of a preset to configure itself.
+
+| Project     | Project Type     | Preset                                                                                                             |
+|-------------|------------------|--------------------------------------------------------------------------------------------------------------------|
+| Application | React            | [`neutrino-preset-react`](https://neutrino.js.org/presets/neutrino-preset-react/)                                  |
+| Application | Web              | [`neutrino-preset-web`](https://neutrino.js.org/presets/neutrino-preset-web/)                                      |
+| Application | Node.js          | [`neutrino-preset-node`](https://neutrino.js.org/presets/neutrino-preset-node/)                                    |
+| Library     | Web              | [`neutrino-preset-taskcluster-web-library`](https://www.npmjs.com/package/neutrino-preset-taskcluster-web-library) |
+| Components  | React Components | [`neutrino-preset-react-components`](https://www.npmjs.com/package/neutrino-preset-react-components)               |
+
+## Test Runners
+Testing is necessary because we all make mistakes. `create-neutrino-project` offers the following test runners:
+
+| Test Runner | Preset                                                                            |
+|-------------|-----------------------------------------------------------------------------------|
+| Jest        | [`neutrino-preset-jest`](https://neutrino.js.org/presets/neutrino-preset-jest/)   |
+| Karma       | [`neutrino-preset-karma`](https://neutrino.js.org/presets/neutrino-preset-karma/) |
+| Mocha       | [`neutrino-preset-mocha`](https://neutrino.js.org/presets/neutrino-preset-mocha/) |
+
+Be sure to check out the test runner preset to get more information on its features and how files should be named.
+
+## Linters
+Linting is the easiest thing you can do to improve a project. The CLI helper offers two linter choices.
```

Comment:
Again, let's try and avoid opinionated speech.

"If you wish to add linting to your project, Neutrino makes it simple to set up during the scaffolding process. `create-neutrino-project` currently offers two linting middleware choices."